reporter/html.js

```diff
@@ -370,15 +374,27 @@ export function escapeText( s ) {
 		label.innerHTML = "Module: ";
 		label.appendChild( moduleSearch );
 
+		applyButton.textContent = "Apply";
+		applyButton.style.display = "none";
+
+		resetButton.textContent = "Reset";
+		resetButton.type = "reset";
+		resetButton.style.display = "none";
+
+		allCheckbox.type = "checkbox";
+		allCheckbox.checked = config.moduleId.length === 0;
+
+		allModulesLabel.classList.add( "clickable" );
```

Comment:
I suspect classList we might still support browsers in QUnit that predate `classList`. Would need to use `className` instead, if so.

Reply:
Didn't expected anyone to still support IE9 and IE10 as even Microsoft has dropped support more that two years ago. But you are right, QUnit webseite says:

> QUnit currently supports the same browsers as jQuery 3.x.

And jQuery lists _Internet Explorer: 9+_.

Pushed the change.